src/lib/usage/Possible.js

```diff
@@ -1,4 +1,7 @@
+// The types towards the beginning require integer min/max limits
 const minMaxTypes = ['str', 'string', 'int', 'integer', 'num', 'number', 'float'];
+// The index of the last type in minMaxTypes that requires integer min/max
+minMaxTypes.intLimitLastIndex = 3;
```

Comment:
Shouldn't this just be a variable instead of adding a property to an array?

Reply:
That would also work. I put it on the array since it was related to it.